src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,20 @@
 package core.basesyntax;
 
 public class MainApp {
+    // In the MainApp class create a Machine array with Truck, Bulldozer,
+    // and Excavator and call methods doWork() and stopWork() in a loop.
+
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Buldozer buldozer = new Buldozer();
+        Excavator excavator = new Excavator();
+
+        Machine[] machineArr = {truck, buldozer, excavator};
```

Comment:
1. not critical but can be simplified
2. better naming
```suggestion
        Machine[] machines = {new Truck(), new Buldozer(), new Excavator()};
```